src/main/scala/gpp/ui/forms/InputEV.scala

```diff
@@ -8,45 +8,44 @@ import japgolly.scalajs.react.component.builder.Lifecycle.RenderScope
 import japgolly.scalajs.react.extra.StateSnapshot
 import japgolly.scalajs.react.vdom.html_<^._
 import japgolly.scalajs.react.{ Callback, ReactEventFromInput, ScalaComponent }
-import monocle.Iso
-import monocle.Prism
 import react.common.ReactProps
 
 /**
-  * Input component that uses a EVar to share the content of the field
+  * Input component that uses a StateSnapshot to share the content of the field
   */
 final case class InputEV[A](
   name:        String,
   id:          String,
   snapshot:    StateSnapshot[A],
-  prism:       Prism[A, String] = Iso.id[String].asPrism,
+  optic:       InputOptics[A] = InputOptics.id,
```

Comment:
Is `InputOptics[A]` shorthand for `Prism[A, String]` or are there differences?

Reply:
in a way it is, but you can construct `InputOptics` not only out of Prism but also out of `Iso` and `Format`

Reply:
Also note that I had the types wrong it was `Prism[A, String]` but it should be `Prism[String, A]`